Models/Courses.py

```diff
@@ -0,0 +1,16 @@
+from sqlalchemy.ext.declarative import declarative_base
+from  sqlalchemy import Column, Integer, String
+
+
+Base = declarative_base()
+
+
+class course(Base):
+    __tablename__ = 'courses'
+    department_ids = Column(String)
```

Comment:
Well, depends on the database. Postgres supports arrays natively and so does SQLAlchemy (http://docs.sqlalchemy.org/en/latest/core/type_basics.html#sqlalchemy.types.ARRAY)